cmd/guacone/cmd/vulnerability.go

```diff
@@ -118,7 +118,7 @@ var queryVulnCmd = &cobra.Command{
 
 				tableRows = append(tableRows, table.Row{vulnResponse.Vulnerabilities[0].Type, vulnResponse.Vulnerabilities[0].Id, "vulnerability ID: " + vulnResponse.Vulnerabilities[0].VulnerabilityIDs[0].VulnerabilityID})
 
-				path, err = queryVulnsViaVulnNodeNeighbors(ctx, gqlclient, pkgResponse, vulnResponse.Vulnerabilities[0].VulnerabilityIDs[0].Id, model.EdgeVulnerabilityCertifyVuln, opts.depth, opts.pathsToReturn)
+				path, err = queryVulnsViaVulnNodeNeighbors(ctx, gqlclient, pkgResponse, vulnResponse.Vulnerabilities, model.EdgeVulnerabilityCertifyVuln, opts.depth, opts.pathsToReturn)
```

Comment:
could you double check that the CLI still works with these changes? - https://docs.guac.sh/querying-via-cli/ else LGTM

Reply:
@nathannaveen all good on this?

Reply:
Sorry for the late response.

I just checked the CLI, and it still works!

Thanks